dockerfiles/README.md

```diff
@@ -26,9 +26,36 @@ Build with (e.g. from the project directory) `docker build ./ -t vulcanize_mcd_t
 NOTE: this file is written for execution on OS X, making use of `host.docker.internal` to access Postgres from the host.
 For execution on linux, replace instances of `host.docker.internal` with `localhost` and run with `--network="host"`.
 
-Running with a local geth client emitting statediffs:
+#### Running with a local geth client emitting statediffs:
 - Use the v1.10-alpha.0 release of the vulcanize go-ethereum patch.
     - this patch allows for statediffs to be emitted
     - start geth with the following flags `--statediff`, `--ws`
 - Make sure to set CLIENT_IPCPATH to use either the websocket or ipc interface with geth (notifications required for subscriptions are not supported with http).
-    e.g. `docker run -e CLIENT_IPCPATH="ws://host.docker.internal:8546" -e DATABASE_NAME="vulcanize_public" -e DATABASE_HOSTNAME="host.docker.internal" -e DATABASE_PORT="5432" -e DATABASE_USER="vulcanize" -e DATABASE_PASSWORD="vulcanize" -e STORAGEDIFFS_SOURCE="geth" vulcanize_mcd_transformers:0.0.1`.
+    e.g. `docker run -e CLIENT_IPCPATH="ws://host.docker.internal:8546" -e DATABASE_NAME="vulcanize_public" -e DATABASE_HOSTNAME="host.docker.internal" -e DATABASE_PORT="5432" -e DATABASE_USER="vulcanize" -e DATABASE_PASSWORD="vulcanize" -e STORAGEDIFFS_SOURCE="geth" -e STARTING_BLOCK_NUMBER=0 vulcanize_mcd_transformers:0.0.1`.
+
+#### Running with a geth docker container:
```

Comment:
Not sure this is necessarily the best place to document this, but wanted to make sure I put it _somewhere_. Happy to move it somewhere else if we can think of a better place!